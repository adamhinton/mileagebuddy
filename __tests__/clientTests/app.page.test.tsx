import Page from "../../src/app/page";

import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import TestReduxStore from "../testutils/clientTestUtils/reduxtestutils/dummyReduxStore";

describe("Page", () => {
	it("renders without errors", () => {
		render(
			<TestReduxStore>
				<Page />
			</TestReduxStore>
		);
	});
});

describe("Sanity check", () => {
	it("2 + 2 = 4", () => {
		const twoPlusTwo = 2 + 2;
		expect(twoPlusTwo).toBe(4);
	});
});
