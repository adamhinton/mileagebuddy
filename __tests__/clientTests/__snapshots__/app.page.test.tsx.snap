// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page matches snapshot from 4.30.2025 1`] = `
<div>
  <main />
</div>
`;
